packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -363,6 +363,26 @@ export const gliner = (model: ModelData): string[] => [
 model = GLiNER.from_pretrained("${model.id}")`,
 ];
 
+
+export const htrflow = (model: ModelData): string[] => [
+	`# HTRflow can be used through CLI or python
+
+# CLI: htrflow pipeline <path/to/pipeline.yaml> <path/to/image>
+# see docs: https://ai-riksarkivet.github.io/htrflow/latest/getting_started/quick_start.html
+ 
+from htrflow.pipeline.pipeline import Pipeline
+from htrflow.pipeline.steps import Task # Choose task
```

Comment:
```suggestion
from htrflow.pipeline.steps import Task
```
I think the quickstart documentation is enough for people to understand how to go about it.